[ANCHOR-704] Document user_action_required_by (#857)
stellar/stellar-docs: platforms/anchor-platform/admin-guide/sep24/configuration.mdx

`SECRET_SEP24_INTERACTIVE_URL_JWT_SECRET` and `SECRET_SEP24_MORE_INFO_URL_JWT_SECRET` are encryption keys that the Anchor Platform will use to generate short-lived tokens it will add to the URLs provided to the wallet. Your business server must also have these keys in its environment so it can verify the token's signature.

> `SEP24_INITIAL_USER_DEADLINE_SECONDS` is the time in seconds that defines the user action deadline before a transaction automatically moves to the next status. It determines the `user_action_required_by` field, which indicates the deadline for statuses with automatic movement. Check [JSON-RPC Methods][json-rpc-methods] for usage examples.

Reviewer: ```suggestion

`SEP24_INITIAL_USER_DEADLINE_SECONDS` is the time in seconds a user has to act before the transaction automatically moves to the next status. It determines the `user_action_required_by` field, which indicates the deadline for statuses with automatic movement. Check [JSON-RPC Methods][json-rpc-methods] for usage examples.
```
